data/lelu.txt

```diff
@@ -1,3 +1 @@
-T | 1 | sleep        
-D | 0 | wake up  | tmr
-E | 1 | sleep  | 9pm  | 10am
```

Comment:
Who sleeps at 9pm? Then wakes up at 10am (which is 11 hours of sleep??). R u ok?